static/src/inline-svgs/journalism/football-weekly-container/play-btnbig.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="34" height="34" viewBox="0 0 34 34">
```

Comment:
Can @akemitakagi or you optimise the SVGs with SVGOMG?

Reply:
👍 I'll do this